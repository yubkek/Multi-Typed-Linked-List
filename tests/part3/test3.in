NEW 3
1
c
32e2
NEW 3
1
2
3
TYPE 0
INSERT 0 0 {1}
TYPE 0
VIEW-NESTED -1
VIEW-NESTED HELLO
VIEW-NESTED 0
