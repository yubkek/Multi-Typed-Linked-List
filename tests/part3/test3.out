List 0: 1 -> c -> 3200.00
List 1: 1 -> 2 -> 3
int -> char -> float
Nested 0: {List 1} -> 1 -> c -> 3200.00
reference -> int -> char -> float
INVALID COMMAND: VIEW-NESTED
INVALID COMMAND: VIEW-NESTED
{1 -> 2 -> 3} -> 1 -> c -> 3200.00